packages/apollo-language-server/src/project/defaultClientSchema.ts

```diff
@@ -0,0 +1,45 @@
+import { GraphQLDocument } from "../document";
+import { Source } from "graphql";
+
+export const apolloClientSchema = `#graphql
```

Comment:
What's the #graphql do? Is that triggering highlighting?

Reply:
Yes it is!